packages/wrangler/templates/middleware/middleware-dispatchee.ts

```diff
@@ -23,7 +23,7 @@ const dispatchee: Middleware = async (request, env, _ctx, middlewareCtx) => {
 	const outboundProxyUrl = request.headers.get(HEADER_OUTBOUND_PROXY_URL);
 	const parameters = request.headers.get(HEADER_PARAMETERS);
 
-	request = new Request(url, { ...request, cf });
+	request = new Request(url, { ...request, cf, method: request.method, headers: request.headers });
```

Comment:
not every prop of a request, it turns out, is enumerable. Being explicit ensure we do pass the headers along.

Reply:
clone doesn't unlock headers, so I'll just stick with this for now :)